postreise/analyze/generation/summarize.py

```diff
@@ -157,18 +168,23 @@ def get_generation_time_series_by_resources(scenario, area, resources, area_type
     return pg.groupby(grid.plant.loc[plant_id, "type"].values, axis=1).sum()
 
 
-def get_storage_time_series(scenario, area, area_type=None):
+def get_storage_time_series(scenario, area, area_type=None, storage_e=False):
     """Get time series total storage energy in certain area of a scenario.
 
     :param powersimdata.scenario.scenario.Scenario scenario: scenario instance
     :param str area: one of *loadzone*, *state*, *state abbreviation*,
         *interconnect*, *'all'*
     :param str area_type: one of *'loadzone'*, *'state'*, *'state_abbr'*,
-    *'interconnect'*
+        *'interconnect'*
+    :param bool storage_e: if set, return time series energy of storage devices
+        instead of power generation. Default to False.
     :return: (*pandas.Series*) -- time series of total storage energy, index: time
         stamps, column: storage energy
     """
     _check_scenario_is_in_analyze_state(scenario)
     storage_id = get_storage_id_in_area(scenario, area, area_type)
 
-    return scenario.state.get_storage_pg()[storage_id].sum(axis=1)
+    if storage_e:
+        return scenario.state.get_storage_e()[storage_id].sum(axis=1)
+    else:
+        return scenario.state.get_storage_pg()[storage_id].sum(axis=1)
```

Comment:
Would it be useful to return both as a `tuple` no matter what, i.e., no need to have the `storage_e` optional parameter.

Reply:
It would take more time especially in a storage heavy scenario. That's the only concern though.

Reply:
It would take more time to... download? read? sum? Even in our heaviest storage scenarios, we have 100x less storage than branches.

Reply:
@danielolsen Theoretically, everything, but you are right, that's minor. However, in a future storage heavy scenario, we might have hundreds of distributed storage devices (maybe?) and in that case, some function calls only need `storage_pg` (such as plot ts generation stack plot), it could improve performance and reduce overhead. Anyway, I'm happy to refactor it into any form as long as you guys are happy:)

Reply:
I think it is fine either way, as long as the docstring is clear.

Reply:
Done.